lib/src/standalone.dart

```diff
@@ -145,6 +145,18 @@ void addStandaloneTasks() {
   addTask(GrinderTask('pkg-standalone-all',
       description: 'Build all standalone packages.',
       depends: tasks.map((task) => task.name)));
+
+  // Add a task for current sdk if it is experimental.
+  var taskNameForCurrentAbi = "pkg-standalone-$_currentOs-$_currentArch";
```

Comment:
It's probably a good idea to mention in `doc/standalone.md` that there will always be a target for the Dart SDK's OS/arch available locally, but it won't necessarily be available on all users' machines.